src/components/felles/enkeltInfo.tsx

```diff
@@ -1,10 +1,11 @@
 import EMDASH from '../../utils/emdash';
 import { StringOrNothing } from '../../utils/felles-typer';
 import { Alert, BodyShort } from '@navikt/ds-react';
+import React from 'react';
 
 interface Props {
     header: string;
-    value?: StringOrNothing;
+    value?: StringOrNothing | React.ReactNode;
```

Comment:
Why ReactNode here, its not so usual to have that in a _value_?

Reply:
src/components/registrering/sporsmolvisning.tsx in this file i need to send in more than a string or -. I can skip using EnkeltInfo if you thinkthat would be better :)

Reply:
I suppose it can be done bit differently in `sporsmolvisning`, enkeltInfo should be probably as it name suggest: enkelt...

Reply:
I agree :) I will fix 👍